library/src/main/java/com/chuckerteam/chucker/internal/support/FileFactory.kt

```diff
@@ -3,5 +3,6 @@ package com.chuckerteam.chucker.internal.support
 import java.io.File
 
 internal interface FileFactory {
-    fun create(): File
+    fun createFileForResponseBody(): File
+    fun createFileForExport(): File
```

Comment:
I would be against such methods. They are too specific and couple responsibilities of separate functionalities. What if there is a need for more file types? Add next method? I think it would be better to provide another implementation of the previous interface and use it where it is needed.

Reply:
Agree. Maybe @adb-shell you can reuse the `create()` method and accept a `fileName : String` as input?